scala-package/mxnet-demo/README.md

```diff
@@ -0,0 +1,60 @@
+# MXNet Scala Sample Project
+This is an project created to use Maven-published Scala package with two Scala examples.
+## Setup
+User are required to use `mvn package` to build the package,
+ which are shown below:
+```Bash
+export SCALA_VERSION_PROFILE=2.11 SCALA_VERSION=2.11.8 MXNET_VERSION=1.2.0
+export SCALA_PKG_PROFILE=
+mvn package -Dmxnet.profile=$(SCALA_PKG_PROFILE) \
+		-Dmxnet.scalaprofile=$(SCALA_VERSION_PROFILE) \
+		-Dmxnet.version=$(MXNET_VERSION) \
+		-Dscala.version=$(SCALA_VERSION)
+```
+These environment variable (`SCALA_PKG_PROFILE`, `SCALA_VERSION_PROFILE`, `MXNET_VERSION`, `SCALA_VERSION`)
+should be set before executing the line above. 
+ 
+You can also use the `Makefile` as an alternative to do the same thing. Simply do the following:
+```Bash
+make scalademo
+```
+This will load the default parameter for all the environment variable.
+ If you want to run with GPU on Linux, just simply add `USE_CUDA=1` when you run the make file
+
+## Run
+### Hello World
+The Scala file is being executed using Java. You can execute the helloWorld example as follows:
+```Bash
+java -Xmx8G  -cp $CLASSPATH sample.HelloWorld
+```
+However, you have to define the Classpath before you run the demo code. More information can be found in the `demo.sh` And you can run the bash script as follows:
+```Bash
+bash bin/demo.sh
+```
+It will load the library automatically and run the example
+### Image Classification using Inference API
```

Comment:
If we are just copying the entire ImageClassification Example, we don't need to maintain it in two different places. I think it is redundant, lets remove this and just keep the HelloWorld
I was thinking of a more self contained code which downloads and runs the model on a fixed image.

Reply:
Your thinking now come true :)

Reply:
thanks, you rock 👍